build-image/openhabian.conf

```diff
@@ -12,8 +12,10 @@ system_default_locale="en_US.UTF-8"
 #timezone=Europe/Berlin
 
 # Wi-Fi settings. An ethernet connection is recommended
-# Requirement: RPi3, RPi0W, PineA64 or a supported external Wi-Fi dongle
+# Requirement: RPi4, RPi3, RPi0W or a supported external Wi-Fi dongle
```

Comment:
~~Ok to state RPi4 here, as it would be rolled out not before we publish a new image....
In other documents I would like to see it running before we put it as supported.~~
Maybe PineA64 should be kept (and removed in the corresponding PR which removed Pine support)

Reply:
No as a PINE already is not guaranteed to work